utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/array/IntervalArray.scala

```diff
@@ -43,7 +43,7 @@ object IntervalArray extends Serializable {
 
   /**
    * Sorts the RDD and collects it to build an IntervalArray, a sorted array that searches
-   * over ranges. This is used for a left side of the broadcast region join in ADAM.
+   * over Intervals. This is used for a left side of the broadcast Interval join in ADAM.
```

Comment:
Intervals &rarr; intervals, etc.

Interval only needs to be capitalized when referring directly to the class.